### ISSUE TITLE
[Bug] Concurrent access fails

### ISSUE THREAD

**Issue by To-om** [TheHive-Project/TheHive#1570]
### Request Type
Bug

### Work Environment
TheHive 3.5.0-RC1
ElasticSearch 7.8.1

### Problem Description
At the end of a Cortex job, the update of the observable fails because of concurrent access.
```
Failed: 1: internal versioning can not be used for optimistic concurrency control. Please use `if_seq_no` and `if_primary_term` instead
[warn] c.c.s.CortexAnalyzerSrv - Unable to insert summary report in artifact
```

### Complementary information
ElasticSearch 7 changes the method of optimistic concurrency control, from version to [sequence number and primary term](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/optimistic-concurrency-control.html#optimistic-concurrency-control)

**Comment by To-om** [comment]
Fixed by TheHive-Project/elastic4play#96

**Comment by nadouani** [comment]
I tested the fix and it's working.
